scripts/sct_deepseg_lesion.py

```diff
@@ -190,6 +190,9 @@ def deep_segmentation_MSlesion(fname_image, contrast_type, output_folder, ctr_al
     tmp_folder = sct.TempFolder()
     tmp_folder_path = tmp_folder.get_path()
     fname_image_tmp = tmp_folder.copy_from(fname_image)
+    if os.path.isfile(ctr_algo):  # if the ctr_algo is a manual centerline file
+        ctr_algo = os.path.basename(ctr_algo)
```

Comment:
why take the base name?

Reply:
For the same reason as:
https://github.com/neuropoly/spinalcordtoolbox/blob/master/scripts/sct_deepseg_lesion.py#L189
--> If we pass the if condition, then the `ctr_algo` is the filename of a nifti mask.

Do you recommend to change the name of the variable (initially this variable reffered to algorithm name)? to change the if condition (ie check if it is a nifti image)?

Reply:
I just want to know why you are trimming the path to its basename, why not keep a full path or absolute path. What if the file is in another folder?
Note: I just took like 1 minute to do a very quick review your code, I didn't look at the implications, just asking.

Reply:
hehe! very good point thank you.
I swap the two lines (see commit ccd068a) : now I copy the file in the tmp folder then I extract the basename --> which will be usefull in the remaining of the code when processing in the tmp_folder.